noir-projects/noir-contracts/contracts/contract_class_registerer_contract/src/events/class_registered.nr

```diff
@@ -14,9 +13,12 @@ pub struct ContractClassRegistered {
     packed_public_bytecode: [Field; MAX_PACKED_PUBLIC_BYTECODE_SIZE_IN_FIELDS],
 }
 
-impl Serialize<MAX_PACKED_PUBLIC_BYTECODE_SIZE_IN_FIELDS + 5> for ContractClassRegistered {
-    fn serialize(self: Self) -> [Field; MAX_PACKED_PUBLIC_BYTECODE_SIZE_IN_FIELDS + 5] {
+impl ContractClassRegistered {
+    fn serialize_non_standard(
+        self: Self,
+    ) -> [Field; MAX_PACKED_PUBLIC_BYTECODE_SIZE_IN_FIELDS + 5] {
         let mut packed = [0; MAX_PACKED_PUBLIC_BYTECODE_SIZE_IN_FIELDS + 5];
+        // Since we are not yet emitting selectors we'll use this magic value to identify events emitted by the ClassRegisterer.
```

Comment:
Copied this comment from constants.nr since I had no idea WTF is this.

Reply:
That's fine, it likely alludes to 'event emission' not really being a thing yet, and event selectors not being a formal concept.